rebar.config

```diff
@@ -4,7 +4,21 @@
 ]}.
 
 {profiles, [
+    {emulation, [
+        {deps, [
+        {grisp_emulation, {git, "git@github.com:grisp/grisp_emulation.git"}},
+        {mapz, "0.3.0"}
```

Comment:
This dependency does not need to be here if it is already in the top level dependency.